newIDE/app/src/GameDashboard/GamesList.js

```diff
@@ -97,19 +178,20 @@ export const GamesList = ({
 
   return (
     <ColumnStackLayout noMargin>
-      <GameRegistration
-        project={project}
-        hideIfRegistered
-        hideLoader
-        onGameRegistered={loadGames}
-      />
+      {!isGameRegistering && (
+        <GameRegistration
+          project={project}
+          hideLoader
+          onGameRegistered={loadGames}
+        />
+      )}
       {displayedGames.map(game => (
         <GameCard
           key={game.id}
           isCurrentGame={!!projectUuid && game.id === projectUuid}
           game={game}
           onOpenGameManager={(tab: GameDetailsTab) => {
-            setOpenedGameInitialTab(tab);
+            addArguments({ 'games-dashboard-tab': tab });
```

Comment:
If I understand well, this should be something like a `navigateTo`? Meaning: does the argument change the tab or is it just to keep the url up to date?

Reply:
the argument actually changes the tab, as you understand it.
So this makes sense to rename this to "navigateTo" indeed, or something similar